### ISSUE TITLE
Make instructions box into its own plugin

### ISSUE THREAD

**Issue by willzeng** [willzeng/WikiNets#175]
Right now the instructions/message box is dependent on the TopBarCreate plugin, which certainly isn't the way to do. 

Should isolate this into its own plugin.

**Comment by backens** [comment]
What are we going to do about different plugins overwriting each other's messages? Put in some sort of priority system? E.g. if someone's creating a link, the "select target node" message should probably not be overwritten by stuff like "double click to find connections", because that's irrelevant and confusing in that moment.

**Comment by willzeng** [comment]
Good point.  Does it make sense to just give each message an integer priority level? This way it works sort of like z-indexes.  

The message is only replaced if it has a higher priority level then what is currently displayed.

Are there other suggestions about how to implement this?

**Comment by vpontis** [comment]
I would say that even an integer system might be too complicated. We see
that problem with z-indices when people hack up a solution and want their
thing in the front they set the z-index to 5000, then the next person sets
the z-index to 9999. It becomes some sort of war.

I would advocate trying to do without a priority system or if we have a
priority system just have it be two-tiered: with priority and without. I'm
sure this does not need a complicated priority system.

## 

Victor Pontis
Massachusetts Institute of Technology 2015
vpontis@mit.edu | 858-761-5232

**Comment by willzeng** [comment]
My vote is for the two level system you suggest Victor.
